src/GitHub.VisualStudio.UI/Views/GitHubPane/PullRequestAnnotationsView.xaml.cs

```diff
@@ -18,5 +17,21 @@ public PullRequestAnnotationsView()
         {
             InitializeComponent();
         }
+
+        IVisualStudioBrowser GetBrowser()
```

Comment:
I think rather than using a global service, you could do the following:
```cs
[Import]
public IVisualStudioBrowser Browser { get; set; }
```

You can then use `Browser.OpenUrl(uri);`.